src/styling/ProjectsPage.css

```diff
@@ -1,101 +1,232 @@
 /** Multiple card view **/
 
-.pg-container {
+.projects-page {
```

Comment:
veryyyy nitpicky, but we can change `projects-page` to an id instead of class since that's more fitting

Reply:
gonna keep this since all the other page containers are classes but will fix the rest ty!